client/state/action-log/index.js

```diff
@@ -0,0 +1,94 @@
+/** @format */
+
+/**
+ * Action Log Redux store enhancer
+ *
+ * Inject at the bottom of the `createStore` enhancer
+ * chain in order to provide access to dispatched actions
+ * as well as the state and store directly from the console.
+ *
+ * Will only attach if the `window` variable is available
+ * globally. If not it will simply be an empty link in the
+ * chain, passing straight through.
+ */
+
+const state = {
+	actionHistory: [],
+	shouldRecordActions: true,
+	historySize: 100,
+};
+
+export const queryToPredicate = query => {
```

Comment:
If we're moving this one, let's move along its tests from `state/console-dispatch/test/*` too

Reply:
Whoops bac2d5decf93063052a9c41f84bf6e877c91418d